### ISSUE TITLE
[FEAT]: Search for actors/actresses

### ISSUE THREAD

**Issue by dschulten** [nova-video-player/aos-AVP#1142]
### Description

Search is not looking for actor/actress names in info  from imdb. It mainly looks for movie titles, as it seems.
Sometimes I want to watch a movie with specific cast members, or check my collection.
For that, cast search would come in handy.

**Comment by phhusson** [comment]
Yeah I think that's a good point. I was starting to do AI shenanigans, but it's possible to start with a simpler full text search.

Here is a quick and dirty demo of this:
Demo code at https://github.com/nova-video-player/aos-Video/tree/dev/phh/full-text-search-demo

https://github.com/nova-video-player/aos-AVP/assets/3642/0e988a71-67fd-476c-85a7-f492f5d26210

It's not completely trivial to integrate properly though, because it requires letting go of system sqlite for our own sqlite, or to have two different sqlite databases (running different versions of sqlite) synchronized between each other. My current demo pushes the full-text-search database in RAM and re-generates it at every app boot, so very far from ideal.

**Comment by dschulten** [comment]
Also, search for directors would be useful: "find Hitchcock movies"
